### DISCUSSION TITLE
[Bug]: cannot drag/pan the video in tracked object details popup on firefox

### DISCUSSION THREAD

**Discussion by ivanjx** [blakeblackshear/frigate#14569] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

on firefox (and zen) i can zoom the video but i cant drag or pan the video control and it plays/pauses the video instead in the tracked object details page.

### Steps to reproduce

1. go to /explore
2. choose an event
3. open the video tab
4. try to zoom and pan the video control

### Version

latest dev

### In which browser(s) are you experiencing the issue with?

Firefox 131.0.3

### Frigate config file

```yaml
irrelevant
```

### docker-compose file or Docker CLI command

```yaml
irrelevant
```

### Relevant Frigate log output

```shell
irrelevant
```

### Relevant go2rtc log output

```shell
irrelevant
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

irrelevant

### Screenshots of the Frigate UI's System metrics pages

irrelevant

### Any other information that may be helpful

the client is ubuntu wayland with firefox installed via flatpak.

**Comment by hawkeye217** [comment]
Thanks, fixed in https://github.com/blakeblackshear/frigate/pull/14587

When a video element is displaying the browser's native video controls, the pan/pinch/zoom is now disabled. The browser's zooming features can still be used.

**Reply by ivanjx** [reply]
can it be made like the live or history video control so i can still pan and zoom?

**Reply by NickM-27** [reply]
No, the user needs to be able to control the video and there is no timeline to be shown here to allow changing the time you are viewing

**Reply by NickM-27** [reply, marked as answer]
if you want to do that just use the review button in the top right to jump to that time in the history page

**Reply by hawkeye217** [reply]
That will remove the player's scrubbing bar, so without the Frigate timeline anywhere, there would be no way to scrub the video. Your browser's native zoom feature should still work fine. You can also download the video and zoom on your device's player.

**Reply by hawkeye217** [reply]
@NickM-27 beat me to it :)
